packages/alea-frontend/pages/api/nap/edit-answer.ts

```diff
@@ -0,0 +1,19 @@
+import { UpdateAnswer } from '@stex-react/api';
+import { NextApiRequest, NextApiResponse } from 'next';
+import {
+  checkIfPostOrSetError,
+  executeAndEndSet500OnError,
+  getUserIdOrSetError,
+} from '../comment-utils';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (!checkIfPostOrSetError(req, res)) return;
+  const userId = await getUserIdOrSetError(req, res);
+  const { answer, id } = req.body as UpdateAnswer;
+  await executeAndEndSet500OnError(
```

Comment:
We are going to limit editing of answers for only some time, right?
Let's add a todo here